spec/models/file_depot_swift_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe FileDepotSwift do
+  let(:uri) { "swift://server.example.com/bucket" }
+  let(:merged_uri) { "swift://server.example.com:5678/bucket?region=test_openstack_region&api_version=v3&domain_id=default" }
+  let(:merged_default_uri) { "swift://server.example.com:5000/bucket?region=test_openstack_region&api_version=v3&domain_id=default" }
+  let(:file_depot_swift) { FileDepotSwift.new(:uri => uri) }
+  it "should require credentials" do
+    expect(FileDepotSwift.requires_credentials?).to eq true
+  end
+
+  it "should return a valid prefix" do
+    expect(FileDepotSwift.uri_prefix).to eq "swift"
+  end
+
+  context "valid merged uri" do
```

Comment:
This should really be the name of the method under test in this block. Also I prefer `describe` for blocks which are testing a particular method and use `context` for blocks which define shared setup.

So this would be the convention we normally would use here:

```ruby
describe "#merged_uri" do
```